app/src/pages/inside/launchesPage/launchesPage.jsx

```diff
@@ -157,6 +165,15 @@ export class LaunchesPage extends Component {
       data: { onConfirm: () => this.moveToDebug(item.id) },
     });
   };
+
+  finishForceLaunches = (eventData) => {
+    let launches = [];
+    eventData.id ? (launches = [eventData]) : (launches = this.props.selectedLaunches);
```

Comment:
`const launches = eventData && eventData.id ? [eventData] : this.props.selectedLaunches`

Reply:
Done